pkg/Microsoft.NETFramework.Compatibility/externalIndex.json

```diff
@@ -0,0 +1,34 @@
+{
+  "Packages": {
+    "System.ServiceModel.Primitives": {
+      "StableVersions": [
```

Comment:
Note: if you didn't want to apply baseline/pre-release to your dependencies you could have modified this: https://github.com/dotnet/corefx/blob/cf28f5105ac4bbcb9d19fa71a90fe4a219e0994b/pkg/dir.targets#L8